### ISSUE TITLE
[Feature request] AppImageUpdate support

### ISSUE THREAD

**Issue by Depaulicious** [alexrj/Slic3r#4315]
It would be nice to have support for https://github.com/AppImage/AppImageUpdate/ in the AppImage for easier upgrade.
#4112 might be required for it to work.

**Comment by lordofhyphens** [comment]
What's the actual benefit here? Users updating their AppImage builds from their desktops?

**Comment by Depaulicious** [comment]
Yes. With AppImageUpdate users can simply run `AppImageUpdate <Slic3r.AppImage>` to get the latest version without looking it up on GitHub.
